src/NServiceBus.Transport.SQS/InputQueuePump.cs

```diff
@@ -41,10 +42,31 @@ class InputQueuePump : IMessageReceiver
             Subscriptions = subscriptionManager;
 
             this.coreSettings = coreSettings;
+            this.setupInfrastructure = setupInfrastructure;
         }
 
         public async Task Initialize(PushRuntimeSettings limitations, OnMessage onMessage, OnError onError, CancellationToken cancellationToken = default)
         {
+            try
+            {
+                inputQueueUrl = await queueCache.GetQueueUrl(ReceiveAddress, cancellationToken).ConfigureAwait(false);
+            }
+            catch (QueueDoesNotExistException ex)
+            {
+                var msg = $"Queue `{ReceiveAddress}` doesn't exist. Call endpointConfiguration.EnableInstallers() to create the queues at startup, or create them manually.";
+                if (setupInfrastructure)
+                {
+                    msg = $"Queue `{ReceiveAddress}` doesn't exist. Ensure this process has the required permissions to create queues on Amazon SQS.";
+                }
```

Comment:
We can do something like this:

```suggestion
                var msg = setupInfrastructure
                    ? $"Queue `{ReceiveAddress}` doesn't exist. Ensure this process has the required permissions to create queues on Amazon SQS."
                    :  $"Queue `{ReceiveAddress}` doesn't exist. Call endpointConfiguration.EnableInstallers() to create the queues at startup, or create them manually.";
```